inventories/README.md

```diff
@@ -10,6 +10,33 @@ We recommend to have 3 kinds of inventories:
 - The OpenVSwitch topology inventory, where you will describe the different bridges and ports configuration for OVS: a template is provided (seapath_ovstopology_definition_example.yml)
 - The VM inventory, where you will describe all variables related to the virtual machine (number of cores, features, ip, network interface, etc.): a template is provided (seapath_vm_definition_example.yml)
 
+## Basic cluster inventory
+
+A basic template of inventory to form a basic cluster for POC is provided (seapath_basic_cluster_definition_example.yml)
+
+#### ⚠ Do not support ⚠
```

Comment:
What is the purpose of this section? Some explanation are needed

Reply:
Corrected by last commit